scripts/pylintrc

```diff
@@ -59,6 +59,8 @@ score=no
 disable=all
 # Identifiers are in the same order as in 'pylint3 --list-msgs'
 enable=
+    fatal,
```

Comment:
Should be put so that it appears where `pylint --list-msgs` lists it, for consistency.

Reply:
Thanks, you finally made me realize something important! I've been confusing `fatal=F001` with `F` the whole time. Now my excuse is: as of pylint 2.1.1 they behave exactly the same because there is no F002, F003,... at least not yet.

This explains why `E` works while `error` doesn't exist.

So here's my latest proposal:
- change `fatal` to `F # atal`
- keep it first because it's not just ant error appearing in `--list-msgs` but an "entire" level
- remove `# E` because this hint is not needed any more

Thoughts?

Reply:
Sounds good to me, as long as the Zephyr repo doesn't trigger any F stuff.

Reply:
Could update the comment above `enable=` to mention that categories are listed first too. Something like

```
# Identifiers are in the same order as in 'pylint --list-msgs'. Categories of
# errors (like F) are listed first.
```

Hints a bit re. what F is too.